account.go

```diff
@@ -107,6 +107,8 @@ type AccountProtectingMultiSigner interface {
 		targetEpoch uint64,
 		targetRoot []byte,
 		domain []byte) ([]e2types.Signature, error)
+
+	SignGenericMulti(ctx context.Context, data []byte, domain []byte, accounts []Account) ([]e2types.Signature, error)
```

Comment:
Please could you add a comment for this function to explain it?  Also, it probably makes sense to put `accounts` ahead of `data` in the argument list.